app/code/community/Apruve/ApruvePayment/controllers/WebhookController.php

```diff
@@ -88,9 +88,11 @@ protected function _capturePayment($invoiceId)
             if ($invoiceId) {
                 /** @var Mage_Sales_Model_Order_Invoice_Api $iApi */
                 $iApi = Mage::getModel('sales/order_invoice_api');
-                if ($iApi->canCapture()){
+                $invoice = Mage::getModel('sales/order_invoice')->loadByIncrementId($invoiceId);
```

Comment:
I'm not sure how this fixes the issue. The error is coming from the code not recognizing the payment_term.accepted webhook. The code shouldn't be calling this method unless the correct webhook has been sent. 

We should instead figure out why the correct webhook type is not being detected.

Reply:
I believe the call is happening because it recognizes the webhook, and tries to call it 1. on an invoice it doesn't possess, and 2. on an iApi object which doesn't have that canCapture method.